tutorials/frontend/from_tflite.py

```diff
@@ -0,0 +1,190 @@
+
+.. code-block:: bash
+
+    # Get the flatc compiler.
+    # You could go to https://github.com/google/flatbuffers to know how to do it.
+    # And you should make sure flatc compiler is installed successfully.
+    flatc --version
+
+    # Get the TFLite schema.
+    wget https://raw.githubusercontent.com/tensorflow/tensorflow/r1.12/tensorflow/contrib/lite/schema/schema.fbs
+
+    # Generate TFLite package.
+    flatc --python schema.fbs
+
+    # Make PYTHONPATH could find generated TFLite package. For example:
+    export PYTHONPATH=/home/someone/tflite
+
+
+After these steps, you could check TFLite package is installed successfully or not.
+For example: ``python -c "import tflite"``
+"""
+######################################################################
+# Utils for downloading and extracting zip files
+# ---------------------------------------------
+
+def download(url, path, overwrite=False):
+    import os
+    if os.path.isfile(path) and not overwrite:
+        print('File {} existed, skip.'.format(path))
+        return
+    print('Downloading from url {} to {}'.format(url, path))
+    try:
+        import urllib.request
+        urllib.request.urlretrieve(url, path)
+    except:
+        import urllib
+        urllib.urlretrieve(url, path)
+
+def extract(path):
+    import tarfile
+    if path.endswith("tgz") or path.endswith("gz"):
+        tar = tarfile.open(path)
+        tar.extractall()
+        tar.close()
+    else:
+        raise RuntimeError('Could not decompress the file: ' + path)
+
+
+######################################################################
+# Load pretrained TFLite model
+# ---------------------------------------------
+# we load mobilenet V1 TFLite model provided by Google
+model_url = "http://download.tensorflow.org/models/mobilenet_v1_2018_08_02/mobilenet_v1_1.0_224.tgz"
+
+# we download model tar file and extract, finally get mobilenet_v1_1.0_224.tflite
+download(model_url, "mobilenet_v1_1.0_224.tgz", False)
+extract("mobilenet_v1_1.0_224.tgz")
+
+# now we have mobilenet_v1_1.0_224.tflite on disk and open it
+tflite_model_file = "mobilenet_v1_1.0_224.tflite"
+tflite_model_buf = open(tflite_model_file, "rb").read()
+
+# get TFLite model from buffer
+import tflite.Model
+tflite_model = tflite.Model.Model.GetRootAsModel(tflite_model_buf, 0)
+
+######################################################################
+# Load a test image
+# ---------------------------------------------
+# A single cat dominates the examples!
+from PIL import Image
+from matplotlib import pyplot as plt
+import numpy as np
+
+image_url = 'https://github.com/dmlc/mxnet.js/blob/master/data/cat.png?raw=true'
+download(image_url, 'cat.png')
+resized_image = Image.open('cat.png').resize((224, 224))
+plt.imshow(resized_image)
+plt.show()
+image_data = np.asarray(resized_image).astype("float32")
+
+# convert HWC to CHW
+image_data = image_data.transpose((2, 0, 1))
```

Comment:
so tf-lite model itself is in NCHW format?

Reply:
TFLite itself is NHWC, we accept NCHW currently like other converters (for example Tensorflow to CoreML converters). Have done it in TFLite Relay frontend currently transparently. You could refer PR #2365 to see more details.

I will update the docs as your comments

Reply:
Thanks, can we explain this a bit in the comments? Since the layout assumption might change later, we'd better ask users to pay attention.

Besides, would you mind open an RFC since it might deserve a serious design discussion.

Reply:
My note has been added to users for paying attention.

Reply:
RFC: https://github.com/dmlc/tvm/issues/2519